auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -263,11 +276,26 @@ def get_previously_added_regressors(self, bug: dict) -> Set[int]:
 
         return added_regressors
 
+    def was_confirmed(self, bug: dict) -> bool:
+        """Check if the bug was confirmed."""
+
+        for entry in reversed(bug["history"]):
```

Comment:
By looking at history only, we exclude bugs that are filed directly in "NEW" status (e.g. by people with editbugs permissions). Is that an explicit choice?